Dockerfile.build

```diff
@@ -0,0 +1,17 @@
+# Builder image
+FROM node:latest AS builder
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+ENV PATH /usr/src/app/node_modules:$PATH
+ADD src /usr/src/app/src
```

Comment:
All works the way you have it. You could probably do a `ADD . /usr/src/app`
As in your NGINX image you are grabbing only the `dist` folder.  It would make your build image slightly bigger as it would grab the tests etc and just be throwing them on the floor. But could allow grabbing any new stuff we add. 🤷‍♂️